client/my-sites/customer-home/cards/constants.js

```diff
@@ -6,6 +6,7 @@ export const EDUCATION_EARN = 'home-education-earn';
 export const EDUCATION_WPCOURSES = 'home-education-wpcourses';
 export const EDUCATION_FIND_SUCCESS = 'home-education-find-success';
 export const EDUCATION_RESPOND_TO_CUSTOMER_FEEDBACK = 'home-education-respond-to-customer-feedback';
+export const EDUCATION_BLOGGING_BASICS = 'home-education-blogging-basics';
```

Comment:
Let's update these to `home-education-blogging-quick-start`, to align with the actual course being called (see: p1635870290408700/1635868542.407800-slack-C029SB8JT8S).

Reply:
> Let's update these to `home-education-blogging-quick-start`, to align with the actual course being called (see: p1635870290408700/1635868542.407800-slack-C029SB8JT8S).

Added a diff at D69494-code to support this; will update this PR to match once it's merged.